apps/api/src/controllers/applicationController.ts

```diff
@@ -230,11 +248,12 @@ export const submitRevision = async ({ applicationId }: { applicationId: number
 
 		const appStateManager = new ApplicationStateManager(application);
 
+		appStateManager.can(ApplicationStateEvents.submit_dac_revisions);
```

Comment:
Should this be a variable or in the conditional? Just calling `.can`?

Reply:
nice catch. code needs to be used, this is incomplete.